wsf_scraping/spiders/who_iris_spider.py

```diff
@@ -58,15 +56,15 @@ def start_requests(self):
         self.data['rpp'] = self.settings['WHO_IRIS_RPP']
         urls = []
         # Initial URL (splited for PEP8 compliance)
-        base_url = 'http://apps.who.int/iris/simple-search'
-        url = base_url + '?location={location}&query={query}&rpp={rpp}'
-        url += '&sort_by={sort_by}&order={order}'
-        url += '&filter_field_1={filter_field_1}&filter_type_1={filter_type_1}'
-        url += '&filter_value_1={filter_value_1}&filter_field_2=language'
-        url += '&filter_type_2=equals&filter_value_2=en&filternumbits=apply'
+        base_url = 'http://apps.who.int/iris/discover'
+        url = base_url + '?&rpp={rpp}&etal=0&group_by=none'
+        url += '&filtertype_0=dateIssued&filtertype_1=iso'
+        url += '&filter_relational_operator_0=contains'
+        url += '&filter_relational_operator_1=contains'
+        url += '&filter_1=en&filter_0={filter_0}'
```

Comment:
This query-string building might be easier if you construct a dict then use `urlencode` from the standard library. Something like:

```python
query_params = {
    'rpp': rpp,
    'etal': 0
    ...
}
url = base_url + '?' + urlencode(query_params)
```

Also I’m not sure `?&rpp=` is what you want.